docs/api-objects.rst

```diff
@@ -44,6 +44,7 @@ API examples
    gl_objects/pipelines_and_jobs
    gl_objects/projects
    gl_objects/project_access_tokens
+   gl_objects/project_secure_files
```

Comment:
Perhaps we could just have `secure_files` here to follow upstream https://docs.gitlab.com/ee/api/secure_files.html, even though they're only implemented at project level now. Project access tokens were a bit different as they have their own page upstream for some reason :grin:

Reply:
Sure thing. Do you want me to make similar changes to the code paths/code itself?

Reply:
Oh yes, that'd make sense so we don't introduce breaking changes later, thanks!

Reply:
Done.